webpack.config.js

```diff
@@ -86,11 +93,17 @@ module.exports = {
         ]
     },
     plugins: [
-        new CopyPlugin({
-          patterns: [{
-            from: "**/*",
-            context: path.resolve(__dirname, "src", "public")
-            }]
+        new HtmlWebpackPlugin({
+          filename: 'index.html',
+          template: 'src/public/index.html',
+          favicon: 'src/public/favicon.ico',
+          publicPath: '.',
         }),
+        ...(DEPLOY_PATH ? [new HtmlWebpackPlugin({
+          filename: 'index-top.html',
+          template: 'src/public/index.html',
+          favicon: 'src/public/favicon.ico',
+          publicPath: DEPLOY_PATH
+        })] : []),
```

Comment:
I looked in s3 and it seems like the index-top.html file is not being created:
s3://models-resources/sensor-interactive/branch/s3-deploy/
Or trying to access it via a browser:
view-source:https://sensor-interactive.concord.org/branch/s3-deploy/index-top.html

So something must be mis configured here. Perhaps the DEPLOY_PATH variable is not making its way through.